performance/server/pom.xml

```diff
@@ -82,6 +82,13 @@
   <build>
     <plugins>
       <plugin>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <version>${version.maven-deploy-plugin}</version>
```

Comment:
Same as above: remove the `<version>` and add `<groupId>org.apache.maven.plugins</groupId>`